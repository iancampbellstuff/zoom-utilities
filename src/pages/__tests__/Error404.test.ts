import { mount, Wrapper } from '@vue/test-utils';
// code under test
import Error404 from '../Error404.vue';

describe('Error404', () => {
    let wrapper: Wrapper<Error404>;
    beforeEach(() => {
        wrapper = mount(Error404, {
            stubs: ['q-btn', 'router-view']
        });
    });
    test('is a Vue instance', () => {
        expect(wrapper.vm).toBeTruthy();
    });
    it('should render correctly', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
});
