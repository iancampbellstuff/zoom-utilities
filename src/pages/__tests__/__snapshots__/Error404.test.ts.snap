// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 should render correctly 1`] = `
<div
  class="fixed-center text-center"
>
  <div>
    <div
      style="font-size: 30vh;"
    >
      
            404
        
    </div>
     
    <div
      class="text-h2"
      style="opacity: 0.4;"
    >
      
            Oops. Nothing here...
        
    </div>
     
    <q-btn-stub
      class="q-mt-xl"
      color="white"
      label="Go Home"
      no-caps=""
      text-color="blue"
      to="/"
      unelevated=""
    />
  </div>
</div>
`;
